python-modules/chevah-python-test/get_binaries_deps.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
we need a few words describing the use/purpose of this file and how it should be used/called.

Why do we need this bash script and not implement it in Python?